.clang-format

```diff
@@ -86,5 +86,18 @@ SpacesInCStyleCastParentheses: false
 SpacesInParentheses: false
 SpacesInSquareBrackets: false
 Standard: c++11
+StatementMacros:
```

Comment:
Reviewing the different `Macro` options in the clang-format docs, it looks like they did add an `AttributeMacros` block. This seems more appropriate for these ([link](https://clang.llvm.org/docs/ClangFormatStyleOptions.html)).